### PR TITLE
.travis.yml: Enable travis-buddy

### PR DESCRIPTION
**Pull request by manankalra**

### CODE REVIEW

**Comment by manankalra** [comment]
https://github.com/coala/documentation/pull/551#issuecomment-383411395 kind of comments can be disabled by turning on _travis-buddy_ only if the tests fail.

File: .travis.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@ script:
 
 notifications:
   email: false
+  webhooks: https://www.travisbuddy.com/
```
Comment (prnvdixit) [inline .travis.yml]: @manankalra Looks good. :+1: 
I also agree with you that adding "on_success: never" should be there as well. Gitmate automatically approves the PR if all CI passes, so there should ideally be no message in that case. :smile:

**Comment by manankalra** [comment]
@jayvdb 
>pre review

didn't get you

**Comment by prnvdixit** [comment]
@manankalra He just meant to add the changes as suggested by me to the PR - No message if all tests pass 😅

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
ack https://github.com/coala/documentation/pull/551/commits/54b819fc79ce4f34658405387ac90ce1c44cceb0

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**